### DISCUSSION TITLE
Failed to load API definition

### DISCUSSION THREAD

**Discussion by ronimhd** [apiman/apiman#2501] [General]
Dear all,

I got this error if I click "API Documentation" button in Portal, and I do not logged in. If I already logged in, the page will be displayed correctly without error.
![image](https://user-images.githubusercontent.com/25297110/231063628-534f99b4-36de-4d87-b4d5-1e72337150ad.png)
(http://apiman.localhost:8080/portal/swagger/mylab/myhttpbin/1.0?try=false&publicAPI=false)
I use 3.1.1 Final version.
My question is, whether this is bug, or I missed some configuration?

**Comment by volkflo** [comment]
Please see #2502
